src/components/chronometer/chronometer.e2e.ts

```diff
@@ -0,0 +1,48 @@
+import { E2EElement, E2EPage, newE2EPage } from "@stencil/core/testing";
+
+describe("gx-chronometer", () => {
+  let element: E2EElement;
+  let page: E2EPage;
+
+  beforeEach(async () => {
+    page = await newE2EPage();
+    await page.setContent(
+      "<gx-chronometer unit='1' value='3000' interval='1500' max-value-text='Final' max-value='9000'></gx-chronometer>"
+    );
+    element = await page.find("gx-chronometer");
+  });
+
+  const delay = value =>
+    new Promise(resolve => setTimeout(() => resolve(), value));
+
+  it("Chronometer not started", async () => {
```

Comment:
I'd change the tests description. The idea with the `it` function is to describe the expected behavior.
In this case I'd use "Sets the chronometer state to started" or "Starts the chronometer".